src/app/components/pages/PostsIndex.scss

```diff
@@ -199,7 +199,7 @@
 
 .articles {
   font-family: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
-  padding: 0.25em 1em;
+  padding: 0em 1em;
```

Comment:
I would prefer not to include style changes in this PR.  Please submit a seperate PR with screenshots illustrating the change.

Reply:
I'm going to close this PR and open two new ones:
- https://github.com/steemit/condenser/pull/3647
- https://github.com/steemit/condenser/pull/3648